packages/generators/utils/language.ts

```diff
@@ -36,23 +40,60 @@ export function getBabelPlugin(): IModuleRule {
 				],
 			],
 		},
-		test: `${new RegExp(/\.js$/)}`,
+		// tslint:enable: object-literal-sort-keys
+	};
+}
+
+export function getTypescriptLoader(): IModuleRule {
+	return {
+		// TODO migrate tslint
+		// tslint:disable: object-literal-sort-keys
+		test: "/\.tsx?$/",
+		loader: "'ts-loader'",
+		include: ["path.resolve(__dirname, 'src')"],
```

Comment:
`'src'` is hardcoded and it shouldn't. If the user chose a different folder, we should set that one.

Reply:
Good point, I didn't notice, thanks!